### PR TITLE
Define more CPU control registers

### PR DESCRIPTION
**Pull request by vbe0201**
As per title, this defines the `ACTRL_ELx`, `DACR32_EL2`, and `CPACR_EL1` registers.

### CODE REVIEW

File: src/registers/cpacr_el1.rs [hunk 1]
```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Copyright (c) 2018-2022 by the author(s)
+//
+// Author(s):
+//   - Valentin B. <valentin.be@protonmail.com>
+
+//! Architectural Feathre Access Control Register - EL1
```
Comment (andre-richter) [inline src/registers/cpacr_el1.rs]:
```suggestion
//! Architectural Feature Access Control Register - EL1
```

File: src/registers/cpuectrl_el1.rs [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+//! Provides additional implementation-defined configuration and control options
+//! for the core.
+
+use tock_registers::{
+    interfaces::{Readable, Writeable},
+    register_bitfields,
+};
+
+register_bitfields! {u64,
+    pub CPUECTRL_EL1 [
```
Comment (andre-richter) [inline src/registers/cpuectrl_el1.rs]: This doesn't seem to be defined in the ARMv8 ARM?

Comment (vbe0201) [reply src/registers/cpuectrl_el1.rs]: Oops, it seems I mixed in a model-specific register with this one. Reverted.

**Comment by andre-richter** [comment]
Thanks!

Again, like https://github.com/rust-embedded/cortex-a/pull/41 I did not review in depth. Should something be off, we'll do bugfix releases.

**Comment by andre-richter** [comment]
Released as `7.4.0`

**Pull request merged**